bin/cobbler-settings

```diff
@@ -114,15 +116,40 @@ def modify(args: argparse.Namespace) -> int:
     schema = migrations.get_schema(migrations.get_installed_version())
     setting_obj = helper.Setting(args.key, args.value)
     # _schema used due to old version of python3-schema in Leap
-    key_type = schema._schema[setting_obj.key_name]
+    try:
+        key_type = schema._schema[setting_obj.key_name]
+    except KeyError:
+        key_found = False
+        for entry in schema._schema:
+            if isinstance(entry, Optional):
+                if entry._schema == setting_obj.key_name:
```

Comment:
I would appreciate a comment explaining why this loop is needed instead of `_schema[key]`-like access. Today I still remember the reason, but I will forget it in no time.

Reply:
I will gladly add a comment. Sorry for the oversight on my end.